sycl/test-e2e/InvokeSimd/Regression/nbarrier_exec_in_order.cpp

```diff
@@ -41,7 +45,8 @@ constexpr int VL = 16;
 template <int case_num> class KernelID;
 
 template <unsigned Threads, unsigned Size, bool UseSLM>
-ESIMD_INLINE void ESIMD_CALLEE_nbarrier(int localID,
+ESIMD_INLINE void ESIMD_CALLEE_nbarrier(local_accessor<int, 1> LocalAcc,
```

Comment:
This test is marked with "// XFAIL: gpu-intel-pvc"
Can you please add some details explaining what exactly causes the fail on this test? Is it caused by nbarrier handling or something else?

Reply:
This test is fail with vISA LTO enabled and pass without the optimization.